docs/components/concepts/clusters.md

```diff
@@ -0,0 +1,44 @@
+---
+id: clusters
+title: "Clusters"
+description: "Learn more about the clusters available in your Camunda Platform 8 plan."
+---
+
+A cluster is a provided group of nodes that run Camunda Platform 8. By default, Camunda Platform 8 clusters are production-ready.
+
+Professional and Enterprise plan customers can create as many production or development clusters as they want.
+
+The following table shows each plan and available type or size of cluster:
+
+|              | Development | Production - S | Production - M | Production - L |
+| ------------ | ----------- | -------------- | -------------- | -------------- |
+| Free         | \-          | \-             | \-             | \-             |
+| Free Trial   | \-          | X              | \-             | \-             |
+| Professional | X           | X              | \-             | \-             |
+| Enterprise   | X           | X              | X              | X              |
+
+## Production clusters
+
+Production clusters come in three sizes: small (S), medium (M), and large (L). To learn more about the size of cluster best suited for your use case, see our [Best Practices](/components/best-practices/best-practices-overview.md) for more information on [sizing your runtime environment](/components/best-practices/architecture/sizing-your-environment.md#sizing-your-runtime-environment).
```

Comment:
"Production clusters come in three sizes: small (S), medium (M), and large (L)." Could we note this before we reference the table above since it contains information required to understand the table (S, M, L)?

Reply:
With this change, I deleted the Production cluster section.